website/content/en/docs/faq.md

```diff
@@ -70,8 +70,69 @@ USER 1001
 If you aren't sure what dependencies are required, start up a container using the image in the `FROM` line as root. That will look something like this.
 `docker run -u 0 -it --rm --entrypoint /bin/bash quay.io/operator-framework/ansible-operator:<sdk-tag-version>`
 
+## I keep seeing errors like "Failed to watch", how do I fix this?
+
+If you run into the following error message:
+
+```
+E0320 15:42:17.676888       1 reflector.go:280] pkg/mod/k8s.io/client-go@v0.0.0-20191016111102-bec269661e48/tools/cache/reflector.go:96: Failed to watch *v1.ImageStreamTag: unknown (get imagestreamtags.image.openshift.io)
+{"level":"info","ts":1584718937.766342,"logger":"controller_memcached","msg":"ImageStreamTag resource not found. 
+``` 
+
+Then, it means that your Operator is unable to watch the resource This scenario can be faced because the Operator has not the permission [(RBAC)[rbac]] to `Watch` the resource or may the Schema from the API used did not implement this verb. In this way the solution would be grant the permission in the `role.yaml` or when it is not  possible use the [client.Reader][client.Reader] instead of the client provide.
```

Comment:
```suggestion
Then, it means that your Operator is unable to watch the resource. This scenario can be faced because the Operator does not have the permission [(RBAC)[rbac]] to `Watch` the resource, or may be the Schema from the API used did not implement this verb. In this way the solution would be to grant the permission in the `role.yaml`,  or when it is not  possible use the [client.Reader][client.Reader] instead of the client provided.
```